source/features/repo-age.tsx

```diff
@@ -30,21 +30,22 @@ const getFirstCommitDate = cache.function(async (): Promise<string | undefined>
 	}
 
 	if (commitsCount === 1) {
-		return select('.commit-tease-sha + span relative-time')!.attributes.datetime.value;
+		return [select('.commit-tease-sha + span relative-time')!.attributes.datetime.value, commitUrl];
 	}
 
-	const relativeTime = await fetchDom(
+	const commit = await fetchDom(
 		`${getRepoURL()}/commits?after=${commitSha}+${commitsCount - 2}`,
-		'.commit-meta relative-time'
+		'.commit-group .commit'
```

Comment:
Selectors don't have to be super specific unless it's necessary
```suggestion
		'.commit'
```